app/controllers/application_controller.rb

```diff
@@ -1,13 +1,19 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::Base
-  include DeviseTokenAuth::Concerns::SetUserByToken
-
-  protect_from_forgery with: :null_session
+  include Pundit
```

Comment:
I'd move this and the rescue to the ApiController, just so we don't respond with a JSON in case you ever use this not as an API response